packages/material-ui/src/Slider/ValueLabel.js

```diff
@@ -1,30 +1,43 @@
 import * as React from 'react';
-import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import { ValueLabelUnstyled } from '@material-ui/unstyled';
+import { useThemeProps, experimentalStyled } from '../styles';
 
-const styles = (theme) => ({
-  thumb: {
-    '&$open': {
-      '& $offset': {
-        transform: 'scale(1) translateY(-10px)',
-      },
+const overridesResolver = (_, styles) => {
+  const componentName = 'MuiSlider-valueLabel';
```

Comment:
?
```suggestion
  const componentName = 'MuiSliderValueLabel';
```

Reply:
I kept the previous classname for this element to avoid changes in the classnames 
![image](https://user-images.githubusercontent.com/4512430/97689752-58162a00-1a9c-11eb-801b-7d00844926f4.png)